src/App.js

```diff
@@ -1,16 +1,98 @@
-import React from 'react';
-import TodoList from './TodoList';
-import AddTodoForm from './AddTodoForm';
+import { React , useState, useEffect } from 'react';
+import {
+  BrowserRouter,
+  Routes,
+  Route
+} from "react-router-dom";
+import TodoList from './components/TodoList';
+import AddTodoForm from './components/AddTodoForm';
+import PropTypes from "prop-types";
+import Airtable from 'airtable';
 
+var base = new Airtable({apiKey: process.env.REACT_APP_AIRTABLE_API_KEY}).base(process.env.REACT_APP_AIRTABLE_BASE_ID);
```

Comment:
You should never really be using `var` anymore, either use `let` or `const`. We never reassign this `base` variable, so `const` would be better.